src/GADM.jl

```diff
@@ -87,6 +87,16 @@ function get(country, subregions...)
 
     data = country |> download |> dataread
 
+    multipolygon = function(geom)
```

Comment:
You still didn't fix the format here. Can you please use a normal function definition without assignments?